.github/workflows/welcome-contributor.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: welcome-contributor
+on:
+    pull_request:
+        types: ["opened"]
+jobs:
+    tpg:
+        if: github.event.pull_request.author_association == "NONE"
+        runs-on: ubuntu-latest
+        name: "Build Terraform and push downstream"
+        env:
+            GOPATH: /home/runner/work/magic-modules/go
+            GO111MODULE: auto
+        steps:
+            - name: Comment on PR.
+              uses: ndmckinley/comment-on-pr@master
```

Comment:
The switch to GitHub Actions is a good opportunity to stop relying on your personal repos :-)